pkg/podexecutor/staticpod.go

```diff
@@ -77,6 +76,11 @@ func (s *StaticPod) APIServer(ctx context.Context, etcdReady <-chan struct{}, ar
 			args = append(args[:i], args[i+1:]...)
 			break
 		}
+		// This option conflicts with cis benchmark
+		if strings.HasPrefix(arg, "--basic-auth=") {
+			args = append(args[:i], args[i+1:]...)
+			break
+		}
```

Comment:
Weird that this is in a chart integration change. On purpose?

Reply:
its a fix for a bug discovered during testing the chart integration, let me know if I should separate it in a different PR though